packages/api/src/external/carequality/command/cq-directory/create-or-update-cq-organization.ts

```diff
@@ -10,17 +10,15 @@ const cq = makeCarequalityManagementAPI();
 export async function createOrUpdateCQOrganization(orgDetails: CQOrgDetails): Promise<string> {
```

Comment:
the type for this func parameter should be `cqOrgDetailsOrgBizRequiredSchema`, otherwise the new attribute isn't going to be used at all... which leads me to believe this hasn't been tested?

Reply:
Ohhh, I missed that, my bad